integration-tests/src/test/java/tachyon/master/JournalIntegrationTest.java

```diff
@@ -158,13 +157,26 @@ public final void before() throws Exception {
   }
 
   /**
+   * Assume {@link #before} has been called, then we need to first stop the cluster and start the
+   * cluster again after reconfiguring {@link Constants#MASTER_JOURNAL_MAX_LOG_SIZE_BYTES}.
+   *
+   * @param bytes max log size in bytes
+   */
+  private void configureMaxLogSizeAndRestart(int bytes) throws Exception {
```

Comment:
In before(), the cluster is started with mLocalTachyonCluster.start(). I think what you can do is create a TachyonConf conf, and set the variable like you do here, and then call mLocalTachyonCluster.start(conf); in before().

Although, then it means all the tests need the same log size. Is that acceptable for these tests?